setup/discovery_server.md

```diff
@@ -102,8 +104,54 @@ turtlebot4-setup
 - When the Create® 3 chimes, call `ros2 topic list` to view topics
 
 ```tip
-When a network change occurs or a robot ROS service or robot is rebooted, you may need to restart the ros2 daemon to see the changes with the ROS 2 command line tools (ros2cli). 
-You may also need to call `ros2 topic list` twice to get a full list of topics. This is because the first time that you run the command, it starts the daemon which will record all of the available topics. If there are a lot of ROS nodes then it may take some time before the full topic list is available. 
+When a network change occurs or a robot ROS service or robot is rebooted, you may need to restart the ros2 daemon to see the changes with the ROS 2 command line tools (ros2cli).
+You may also need to call `ros2 topic list` twice to get a full list of topics. This is because the first time that you run the command, it starts the daemon which will record all of the available topics. If there are a lot of ROS nodes then it may take some time before the full topic list is available.
```

Comment:
Superfluous new line or maybe this should be two paragraphs (flagged because the space was removed as part of that other discussion)

Reply:
Added paragraph break